app/controllers/welcome_controller.rb

```diff
@@ -15,6 +15,13 @@ def popular
     renders(@events)
   end
 
+  def about
```

Comment:
`@attendees`, `@tickets` and `@sponsors` as variable names does not rightly reflect what these variables are, which are counts. Kindly check variable naming.